appveyor.yml

```diff
@@ -1,5 +1,7 @@
 version: "{build}"
 
+skip_branch_with_pr: true
```

Comment:
Prevents a duplicate build when you have a PR open.

![image](https://user-images.githubusercontent.com/12101120/107092158-0b479b00-67d1-11eb-9bcd-9ed5dbc7fa74.png)